Directory.Packages.props

```diff
@@ -1,66 +1,75 @@
-        <PackageVersion Include="Microsoft.AspNetCore.Authentication.JwtBearer" Version="9.0.0" />
-        <PackageVersion Include="Microsoft.AspNetCore.OpenApi" Version="9.0.0" />
-        <PackageVersion Include="Microsoft.Azure.Functions.Worker" Version="2.0.0" />
-        <PackageVersion Include="Microsoft.Azure.Functions.Worker.ApplicationInsights" Version="2.0.0" />
-        <PackageVersion Include="Microsoft.Azure.Functions.Worker.Extensions.DurableTask" Version="1.2.0" />
-        <PackageVersion Include="Microsoft.Azure.Functions.Worker.Extensions.Http" Version="3.2.0" />
-        <PackageVersion Include="Microsoft.Azure.Functions.Worker.Extensions.Storage.Queues" Version="5.5.0" />
-        <PackageVersion Include="Microsoft.Azure.Functions.Worker.Extensions.Timer" Version="4.3.1" />
-        <PackageVersion Include="Microsoft.Azure.Functions.Worker.Sdk" Version="2.0.0" />
-        <PackageVersion Include="Microsoft.ClearScript.Complete" Version="7.4.5" />
-        <PackageVersion Include="Microsoft.Data.SqlClient" Version="6.0.1" />
-        <PackageVersion Include="Microsoft.EntityFrameworkCore.Design" Version="9.0.0" />
-        <PackageVersion Include="Microsoft.EntityFrameworkCore.SqlServer" Version="9.0.0" />
-        <PackageVersion Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="9.0.0" />
-        <PackageVersion Include="Microsoft.Extensions.Diagnostics.HealthChecks" Version="9.0.0" />
-        <PackageVersion Include="Microsoft.Extensions.Diagnostics.HealthChecks.EntityFrameworkCore" Version="9.0.0" />
-        <PackageVersion Include="Microsoft.Extensions.Http.Polly" Version="9.0.0" />
-        <PackageVersion Include="Microsoft.Extensions.Logging" Version="9.0.0" />
-        <PackageVersion Include="Microsoft.Extensions.Logging.Abstractions" Version="9.0.0" />
-        <PackageVersion Include="Microsoft.Extensions.Options" Version="9.0.0" />
-        <PackageVersion Include="Microsoft.VisualStudio.Threading.Analyzers" Version="17.13.2" />
-        <PackageVersion Include="Microsoft.NET.Test.Sdk" Version="17.13.0" />
-        <PackageVersion Include="Newtonsoft.Json" Version="13.0.3" />
-        <PackageVersion Include="OpenAI" Version="2.0.0" />
-        <PackageVersion Include="Polly" Version="8.5.0" />
-        <PackageVersion Include="Polly.Contrib.WaitAndRetry" Version="1.1.1" />
-        <PackageVersion Include="ReverseMarkdown" Version="4.6.0" />
-        <PackageVersion Include="SendGrid" Version="9.29.3" />
-        <PackageVersion Include="SendGrid.Extensions.DependencyInjection" Version="1.0.1" />
-        <PackageVersion Include="Swashbuckle.AspNetCore" Version="7.1.0" />
-        <PackageVersion Include="xunit.v3" Version="1.1.0" />
-        <PackageVersion Include="xunit.analyzers" Version="1.20.0" />
-        <PackageVersion Include="xunit.runner.visualstudio" Version="3.0.2" />
-    </ItemGroup>
+  <PropertyGroup>
```

Comment:
I started working on some editor config changes to prevent this whitespace change last week but forgot to get back to it.  For now it should be quick to format this file in Rider to get four space indentation and get rid of most of these line changes.